common/src/main/java/tachyon/util/FormatUtils.java

```diff
@@ -166,5 +167,23 @@ public static long parseSpaceSize(String spaceSize) {
     }
   }
 
+  /**
+   * Formats file permission to human-readable version.
+   *
+   * @param permission file permission
+   * @param isFolder if the Path is folder
+   * @return human-readable version of permission
+   */
+  public static String formatPermission(short permission, boolean isFolder) {
```

Comment:
Got it, I think moving it to `FormatUtils` is better, `UI` calls the method from `CommandUtils` may be a little strange.